integrations.yml

```diff
@@ -395,4 +395,4 @@
   description: "Track time effortlessly with Toggl Button integration for Chrome and Firefox - the simplest time tracker to help you get things done."
   website: "https://www.toggl.com/toggl-button/"
   icon: "toggl.png"
-  category: 2
+  category: 2
```

Comment:
Would you mind dropping this change that removes the `\n` at the end of `integrations.yml`?

Reply:
Sure no problem, removed now 😄

Reply:
Thank you! 🙏